src/styles/star.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
![image](https://user-images.githubusercontent.com/97547824/211389027-16a3a4be-78d2-4903-9b17-f2f88652393d.png)

Reply:
Fix it